internal/pkg/update/client.go

```diff
@@ -68,7 +68,7 @@ func NewClient(params *ClientParams) *client {
 
 // CheckForUpdates checks for new versions in the repo
 func (c *client) CheckForUpdates(cliName, currentVersion string, forceCheck bool) (string, string, error) {
-	if c.DisableCheck {
+	if c.DisableCheck || currentVersion == "v0.0.0" {
```

Comment:
Can a user actually end up with `v0.0.0` somehow?

Reply:
v0.0.0 is the default value if the relevant ldflag isn't passed during the build process. A customer shouldn't ever encounter this, though.

Reply:
Maybe we should make a constant for v0.0.0, since it's not obvious what this means